kolibri/plugins/coach/assets/src/modules/lessonResources/handlers.js

```diff
@@ -226,3 +200,20 @@ function _prepLessonContentPreview(store, classId, lessonId, contentId) {
     }
   );
 }
+
+export function showLessonResourceSearchPage(store, params) {
+  return store.dispatch('loading').then(() => {
+    return ContentNodeSearchResource.fetchCollection({
+      getParams: {
+        search: params.searchTerm,
+      },
+    }).then(results => {
+      return showResourceSelectionPage(store, {
+        classId: params.classId,
```

Comment:
Then would just need to pass `channel_ids`, `content_kinds` and `total_results` to the state and use it to populate the filters. Could potentially add an additional return property from the search for the coach content filter also.